ir.imansalmani.IPlan.json

```diff
@@ -0,0 +1,51 @@
+{
+  "app-id": "ir.imansalmani.IPlan",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "43",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "iplan",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "iplan",
+      "config-opts": ["-Dprofile=default"],
+      "builddir": true,
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "branch": "main",
+          "tag": "1.0.0",
+          "commit": "36e41b5dd3c10e76b0b332ceaef07e29481e50a6",
+          "url": "https://github.com/iman-salmani/iplan.git"
+        }
+      ]
+    }
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "build-args": ["--share=network"],
```

Comment:
the build need to be offline. Either you use vendored crate, or you use the https://github.com/flatpak/flatpak-builder-tools

Also `build-options` is usually above the modules in the manifest.

Reply:
i added dependencies by flatpak-cargo-generator but how its works when cargo runs with --offline option in meson?